merlion/models/forecast/ets.py

```diff
@@ -156,11 +160,20 @@ def _forecast(
 
         # Basic forecasting without time_series_prev
         if time_series_prev is None:
-            forecast_result = self.model.get_prediction(
-                start=self._n_train, end=self._n_train + len(time_stamps) - 1, method="exact"
-            )
-            forecast = np.asarray(forecast_result.predicted_mean)
-            err = np.sqrt(np.asarray(forecast_result.var_pred_mean))
+            # Some variants of ETS model does not support prediction interval when pred_interval_strategy="exact".
+            # In this case we use point forcasting and set prediction_interval as None
+            try:
+                forecast_result = self.model.get_prediction(
+                    start=self._n_train,
+                    end=self._n_train + len(time_stamps) - 1,
+                    method=self.config.pred_interval_strategy,
+                )
+                forecast = np.asarray(forecast_result.predicted_mean)
+                err = np.sqrt(np.asarray(forecast_result.var_pred_mean))
+            except NotImplementedError as v:
+                forecast_result = self.model.predict(start=self._n_train, end=self._n_train + len(time_stamps) - 1)
+                forecast = np.asarray(forecast_result)
+                err = np.full(len(forecast), np.nan)
```

Comment:
Why do you use `np.nan` here instead of `forecast_result.var_pred_mean` as before? This is a more sensible fallback behavior in my opinion.

Reply:
I understand your original rationale of setting err = None better now. I've updated some of the other forecasting code to be more robust to this case.

Reply:
some variants of ETS model do not support probabilistic forecasting, `get_prediction` will raise error here, `predict` only provide point forecasting